ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/version/withoutledgerid/VersionClient.scala

```diff
@@ -22,4 +35,87 @@ private[daml] final class VersionClient(service: VersionServiceStub) {
         new GetLedgerApiVersionRequest(ledgerIdToUse.unwrap)
       )
       .map(_.version)
+
+  def getApiFeatures(
+      ledgerIdToUse: LedgerId,
+      token: Option[String] = None,
+  )(implicit executionContext: ExecutionContext): Future[Seq[Feature]] =
+    LedgerClient
+      .stub(service, token)
+      .getLedgerApiVersion(
+        new GetLedgerApiVersionRequest(ledgerIdToUse.unwrap)
+      )
+      .map(_.features.toList.flatMap(VersionClient.fromProto))
+}
+
+private[daml] object VersionClient {
+  // see also com.daml.platform.apiserver.services.ApiVersionService.featuresDescriptor
+  def fromProto(featuresDescriptor: FeaturesDescriptor): Seq[Feature] =
+    featuresDescriptor match {
+      case FeaturesDescriptor(userManagement, experimentalFeatures) =>
+        (userManagement.toList map (_ => Feature.UserManagement)) ++
+          (experimentalFeatures.toList flatMap fromProto)
```

Comment:
@adriaanm-da : do you need to use these experimental features? They are essentially internal features that we don't want to expose to customers. So, I'm not sure whether we should expose them in the official client. At least not at the same level as the official features. Do you see a good way to provide this distinction at the API level?

Reply:
I guess my intuitive preference is to not expose them at all.

Reply:
I only needed `userManagement`, and then just went for exhaustiveness for no good reason other than to appease scalac :-) 

Happy to lock the API down more -- I hadn't considered the customer angle. How do we make a decision? Who else should I loop in?

Reply:
In general, we can always open up our API's more. Locking them down is the hard part.

As for the decision, I'd like to know what @stefanobaghino-da and @gerolf-da think, and then make a decision based on our input.

My vote currently goes to "do not expose experimental features".